apps/fluent-tester/src/TestComponents/Icon/IconExperimental.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FontIcon, SvgIcon, Icon } from '@fluentui-react-native/experimental-icon';
+import { Text, View } from 'react-native';
+import TestSvg from './assets/test.svg';
+import { SvgIconProps } from '@fluentui-react-native/icon';
+
+const fontBuiltInProps = {
+  fontFamily: 'Arial',
+  codepoint: 0x2663,
+  fontSize: 32,
+};
+
+export const IconExperimental: React.FunctionComponent = () => {
+  const svgProps: SvgIconProps = {
```

Comment:
This const can be created outside the function component so that it's not recreated on every render